### ISSUE TITLE
Level: Bot

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#526]
```
:name=Bot
:description=A new challenger enters
:author_name=andybalaam
:author_url=https://github.com/andybalaam
:num_rabbits=2
:num_to_save=2
:rabbit_delay=2
:bridge=4
:dig=1
:music=tryad-the_final_rewind
##############
#            #
#            #
#            #
#            #
# Q          #
#            #
#      t  tO #
##############
```

https://github.com/andybalaam/rabbit-escape/blob/master/rabbit-escape-engine/src/rabbitescape/levels/staging/rabbots/Bot.rel

**Comment by andybalaam** [comment]
We should move the rabbots back a bit to make this fairer. Too much early reaction speed needed at the moment.

**Comment by andybalaam** [comment]
Released in 0.11
